src/indicator_lib.py

```diff
@@ -0,0 +1,60 @@
+import numpy as np
+
+def calc_ema(dataframe, ema_size):
```

Comment:
It looks like emas are always calculated based off of candles, so let's name dataframe here candles so as to clarify what the dataframe is.